### PR TITLE
styling the plugin editor - issue #37

### PR DESCRIPTION
**Pull request by radfordwill**

### CODE REVIEW

**Comment by radfordwill** [comment]
I can't resolve one file without resolving the other. Should I submit the css file this way? I don't want to mangle anything. My minifying script turns out different than yours.

**Comment by danieltj27** [comment]
Run this command in terminal `sass --watch darkmode.scss:darkmode.css --style compressed` to get the CSS file minified. That way it will not cause such a big change for the CSS that is output. I'll make a review of the changes anyway so you can fix everything at once.

File: darkmode.scss [hunk 1]
```diff
@@ -1150,19 +1152,48 @@ body {
...
 
 				&-comment {
 
 					color: $light-grey;
 
 				}
+				
+				&-keyword {
+
+					color: $lavender;
```
Comment (danieltj27) [inline darkmode.scss]: Need to prefix `editor` to this variable as it's not defined.

File: darkmode.scss [hunk 2]
```diff
@@ -1150,19 +1152,48 @@ body {
...
+				
+				&-def {
 
+					color: $medium-blue;
+
+				}
+				
+				&-variable_2{
+
+					color: $sunglo;
```
Comment (danieltj27) [inline darkmode.scss]: Same as above, needs `editor` prefix.

File: darkmode.scss [hunk 3]
```diff
@@ -1150,19 +1152,48 @@ body {
...
+
+				}
+				
+				&-def {
 
+					color: $medium-blue;
+
+				}
+				
+				&-variable_2{
```
Comment (danieltj27) [inline darkmode.scss]: Add space between class name and bracket.

File: darkmode.scss [hunk 4]
```diff
@@ -1150,19 +1152,48 @@ body {
...
+
+					color: $sunglo;
+
+				}
+				
+				&-string {
+
+					color: $olivine;
```
Comment (danieltj27) [inline darkmode.scss]: Needs the `editor` prefix adding here.

File: darkmode.css [hunk 5]
```diff
@@ -1,2 +1,523 @@
...
+body {
```
Comment (danieltj27) [inline darkmode.css]: See my comment in the PR thread - please minify the CSS output file.

**Comment by radfordwill** [comment]
I see it, thanks for the info. will work it out asap today.

**Comment by radfordwill** [comment]
changes made and submitted

**Comment by danieltj27** [comment]
Can you do a `git pull` from this repo to your and then submit your changes again. It looks like the version you submitted with your new PR code is a couple of iterations behind the current version in this repo. Then I can merge the changes in.

**Comment by radfordwill** [comment]
resolved conflicts manually with latest changed css

**Pull request merged**